### PR TITLE
Done and Undone Commands will add/delete Revenue from Model 

### LINKED ISSUE #133: As a home-based beauty salon owner, I want the revenue attached to a service to be recorded after an appointment is done.

### PR DESCRIPTION
**Pull request by khoongwk**
Made these commands list all appointments after execution too, using model.updateFilteredExpenseList(Model.PREDICATE_SHOW_ALL_APPOINTMENTS);.

Also, some refactoring to privatize local message strings and renamed some instances of Supersalon to Homerce.

### CODE REVIEW

File: src/main/java/seedu/homerce/commons/core/Messages.java [hunk 1]
```diff
@@ -16,6 +17,11 @@
     public static final String MESSAGE_INVALID_APPOINTMENT_DISPLAYED_INDEX = "The appointment"
             + " index provided is invalid";
     public static final String MESSAGE_APPOINTMENTS_LISTED_OVERVIEW = "%1$d appointments listed!";
+    public static final String MESSAGE_APPOINTMENT_ALREADY_DONE = "This appointment is already marked as done!";
+    public static final String MESSAGE_APPOINTMENT_ALREADY_UNDONE = "This appointment is already marked as not done!";
+    public static final String MESSAGE_DUPLICATE_APPOINTMENT = "This appointment already exists in the homerce book.";
```
Comment (hanstirtaputra) [inline src/main/java/seedu/homerce/commons/core/Messages.java]: Maybe using "exists in Homerce" is sufficient instead of "homerce book"?

**Review by hanstirtaputra — COMMENTED** [review]
I think one of the messages be rephrased for consistency

File: src/main/java/seedu/homerce/commons/core/Messages.java (hunk 1 above)
Comment (khoongwk) [reply src/main/java/seedu/homerce/commons/core/Messages.java]: Okay, will rename. Apparently there are many instances of "Homerce book" throughout the project. I think a "find-and-replace" went wrong in the past.

**Review by hanstirtaputra — APPROVED** [review]
Great thanks! LGTM

File: src/main/java/seedu/homerce/commons/core/Messages.java (hunk 1 above)
Comment (hanstirtaputra) [reply src/main/java/seedu/homerce/commons/core/Messages.java]: Yes I think it happened during the refactoring from Addressbook to Homerce

**Pull request merged**